be/src/vec/functions/function_timestamp.cpp

```diff
@@ -115,12 +115,9 @@ struct StrToDate {
         ColumnPtr argument_columns[2] = {
                 col_const[0] ? static_cast<const ColumnConst&>(*col0).convert_to_full_column()
                              : col0};
-        check_set_nullable(argument_columns[0], null_map, col_const[0]);
```

Comment:
for this struct, u can see `static constexpr auto name = "str_to_date";`
so make sure the functions have enough testcases too